src/App.css

```diff
@@ -28,33 +30,32 @@
   color: #61dafb;
 }
 
-.title {
-  padding: 24px 24px 24px 24px;
-}
-
 .container {
   display: flex;
-  height: 30px;
+  height: 48px;
   margin-bottom: 16px;
   text-align: center;
-  padding: 0px 0px 0px 24px;
 }
 
 
 .id-container {
```

Comment:
try to add 
```
display: flex;
align-items: center;
justify-content: center;
```

to this and `.task-name-container`

You can also do
```
.id-container, .task-name-container {
    display: flex;
    align-items: center;
    justify-content: center;
}
```
to add style to both at the same time